### PR TITLE
css-to-ls

### PR DESCRIPTION
**Pull request by kghamilton89**
Refactor ChartLab code blocks to use `ls` dialect instead of `css`.

### CODE REVIEW

File: tutorials/workshop/lets-encrypt.md [hunk 1]
```diff
@@ -1184,7 +1184,7 @@ public class CertTrust {
 
 Validation results for Expired Certificate:
 
-```css
+```ls
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply tutorials/workshop/lets-encrypt.md]: Changed

File: integrations/itm/resources/metric-list.md [hunk 2]
```diff
@@ -563,7 +563,7 @@ kvmcltrsrv.server_cpu_utilization
 
 ## WebSphere MQ
 
-```css
+```ls
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply integrations/itm/resources/metric-list.md]: changed

**Comment by rodionos** [comment]
image

Use `ls` for chart configuration snippets that start with section, e.g.:

```ls
[widget]
   type = pie
```

**Comment by kghamilton89** [comment]
Reviewed diff and corrected errors.

**Comment by kghamilton89** [comment]
Travis is calling a live link dead.

**Pull request merged**